app/models/updateVatSubscription/request/UpdatedOrganisationDetails.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.updateVatSubscription.request
+
+import play.api.libs.json.{Json, Writes}
+
+case class UpdatedOrganisationDetails(organisationName: Option[String],
```

Comment:
Could you create some unit tests for this model to verify the JSON writes functionality.